Formula/pyside.rb

```diff
@@ -40,26 +40,20 @@ def install
   end
 
   test do
-    system Formula["python@3.9"].opt_bin/"python3", "-c", "import PySide2"
-    system Formula["python@3.9"].opt_bin/"python3", "-c", "import shiboken2"
+    system Formula["python@3.9"].opt_bin/"python3", "-c", "import PySide6"
+    system Formula["python@3.9"].opt_bin/"python3", "-c", "import shiboken6"
 
     modules = %w[
       Core
       Gui
-      Location
-      Multimedia
```

Comment:
Are these modules not there in pyside6?

Reply:
See the blog [post](https://www.qt.io/blog/add-on-support-in-qt-6.0-and-beyond)

Reply:
Can you still put them in `modules`, but comment them out, with a note that they can be added in qt6.2? Same for `WebEngineWidgets`, if that will be available in qt6.2.